test/Makefile

```diff
@@ -3,7 +3,7 @@ include ../Make.inc
 
 # Set rpath of tests to builddir for loading shared library
 OPENLIBM_LIB = -L.. -lopenlibm
-ifneq (,$(findstring $(OS),Linux FreeBSD))
+ifneq (,$(findstring $(OS),Linux FreeBSD DragonFly))
```

Comment:
Actually, this should probably just be
```make
ifneq ($(OS),Windows_NT)
ifneq ($(OS),Darwin)
OPENLIBM_LIB += -Wl,-rpath=$(OPENLIBM_HOME)
endif
endif
```
That would likely cover the bases for any other BSD we want to support.

Reply:
Why not, better use the case which is most likely to work for unlisted platforms as a fallback.